sql/sys_vars.cc

```diff
@@ -2597,11 +2597,11 @@ static Sys_var_max_user_conn Sys_max_user_connections(
        NOT_IN_BINLOG, ON_CHECK(if_checking_enabled));
 
 static Sys_var_ulong Sys_max_tmp_tables(
-       "max_tmp_tables", "Unused, will be removed.",
+       "max_tmp_tables", NULL,
        SESSION_VAR(max_tmp_tables), CMD_LINE(REQUIRED_ARG),
```

Comment:
I think the old help text made sense. Someone looking at the config file might want to know what `max_tmp_table=10000` did.